fbfmaproom/Dockerfile

```diff
@@ -12,17 +12,17 @@ RUN rm -r /etc/httpd/conf.d /etc/httpd/conf.modules.d
 
 # miniconda
 RUN curl -L \
-    https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh \
+    https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh \
```

Comment:
Updating to the latest version of conda has nothing to do with the intent of this PR, it's just something I do whenever I modify the Dockerfile so we don't fall too far behind.